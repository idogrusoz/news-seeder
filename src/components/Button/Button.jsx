import React from 'react';

const Button = ({ handleClick, label }) => {
    const style = {
        backgroundColor: '#37e8b8',
        border: 'none',
        color: '#fff',
        padding: '10px 15px',
        cursor: 'pointer',
        margin: '5px',
    };
    return (
        <button onClick={handleClick} style={style}>
            {label}
        </button>
    );
};

export default Button;
