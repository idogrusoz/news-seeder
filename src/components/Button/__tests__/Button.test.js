import { shallow } from 'enzyme';
import Button from '../Button';

describe('Button test', () => {
    it('matches the snashot', () => {
        const wrapper = shallow(<Button handleClick={jest.fn()} label="mock" />);
        expect(wrapper).toMatchSnapshot();
    });
});
