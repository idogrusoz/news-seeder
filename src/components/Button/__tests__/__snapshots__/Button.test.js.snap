// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test matches the snashot 1`] = `ShallowWrapper {}`;
